### PR TITLE
[Infra-2364] Deploy ldap on publick8s

### PR DESCRIPTION
**Pull request by olblak**
This PR is part of [INFRA-1797](https://issues.jenkins-ci.org/browse/INFRA-1797)

### CODE REVIEW

File: charts/ldap/templates/persistentVolumeClaim.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    requests:
+      storage: 100Gi
+  selector:
+    matchLabels:
+      role: ldap-backup
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "ldap.fullname" . }}-data
```
Comment (timja) [inline charts/ldap/templates/persistentVolumeClaim.yaml]: I can't find the PV for this in the PR? is it defined somewhere else?

Comment (olblak) [reply charts/ldap/templates/persistentVolumeClaim.yaml]: This is the default behavior, Kubernetes provisions a new azure disk storage and then attach it to the pod, which means that the lifecycle is tied to kubernetes cluster.

image

More information on this application design [here](https://github.com/jenkins-infra/iep/tree/master/iep-008#data)

**Comment by olblak** [comment]
> README for the chart?

Good point

File: charts/ldap/templates/persistentVolumeClaim.yaml (hunk 1 above)
Comment (timja) [reply charts/ldap/templates/persistentVolumeClaim.yaml]: Ah, I see. thanks for the link, didn't realise the backup/restore was so fast

File: charts/ldap/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- [Docker](https://github.com/jenkins-infra/ldap)
+- [DockerHub](https://hub.docker.com/r/jenkinsciinfra/ldap)
+- [IEP](https://github.com/jenkins-infra/iep/tree/master/iep-008)
+
+
+## Running this yourself
+
+`helmfile -f helmfile.d/ldap.yaml apply`
+
+ ### Minikube gotchas
```
Comment (timja) [inline charts/ldap/README.md]: extra space was there:
```suggestion
### Minikube gotchas
```

Comment (olblak) [reply charts/ldap/README.md]: thx

**Pull request merged**